modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -654,6 +655,35 @@ enum { CAP_PROP_IMAGES_BASE = 18000,
 
 //! @} Images
 
+/** @name OBSENSOR (for Orbbec 3D-Sensor device/module )
+    @{
+*/
+//! OBSENSOR data given from iamge generator
+enum { 
+    CAP_OBSENSOR_DEPTH_MAP = 0, //!< Depth values in mm (CV_16UC1)
+    CAP_OBSENSOR_BGR_IMAGE = 1, //!< Data given from BGR stream generator
```

Comment:
BGR image is preserved to default stream 0 (of any VideoCapture backend)

Reply:
According to the existing code, both OpenNI and Intel Realsense use depth as the default stream。For RGB-D cameras, depth as the default stream is more in line with user needs as they may only need the depth data stream

Reply:
@vpisarev Vadim, could you take a look on this and put a comment?

Reply:
@alalek, @hzcyf, I can confirm that for OpenNI and Intel RealSense (PerC) cameras depth map is the default stream. From this point of view the new Orbbec API follows this tradition. I think, it should be retained as-is.

Reply:
Problem is that this support is enabled by default (for many build configurations).
Existed user code may not expect non 8UC3 data and crash (16SC1/16UC1 or 32FC1 are used for depth).
Exception is preferable (and reliable for this case) than silent replacing of 8UC3 result.

If we go in this way, then we must update many sample code / documentation snippets / etc. But this would not resolve problem with existed users code.

BTW, another approach may go through another API, like `VideoDepthCapture`. Including OpenNI/Realsense "buggy" cases.

Reply:
@alalek, there is already such situation with OpenNI and PerC and so far it did not cause any major problems. Let me merge this PR